.github/workflows/acceptance-workflow.yml

```diff
@@ -38,12 +38,8 @@ jobs:
         run: npx lerna run build
 
       - name: Run acceptance tests
-        continue-on-error: true
-        timeout-minutes: 15
-        run: npm run acceptancetest:${{ inputs.testfilter }}
-        
-      - name: Re-run acceptance tests (without nick-fields/retry@v2)
-        if: ${{ failure() }}
-        timeout-minutes: 15
-        run: npm run acceptancetest:${{ inputs.testfilter }}
-
+        uses: nick-fields/retry@v2
```

Comment:
Glad to see this works now.
Previously the tests would never end and so we couldn't use it.

Reply:
Yep, now they're retrying 3 times before showing fail.